Assignment 1/Part 4 - Player/Player.cpp

```diff
@@ -16,20 +16,23 @@ using namespace std;
 
 // Player Class
 
-
 Player::Player(){ //Default constructor create empty players
-
+	players++;
+	playerID = players;
 };
 
 Player::~Player()
 {
-	
+	players--;
```

Comment:
you shouldn't do this, players is the number of created Players. So it should never decrease or it will 100% of the time crate duplicates!

Reply:
I also think that we'll have to implement the observer somewhere else, in one of the new classes Simon and I created